crypto/multisig/src/lib.rs

```diff
@@ -122,6 +129,11 @@ impl<P: Default + PublicKey + Message> SignerSet<P> {
     where
         P: Verifier<S>,
     {
+        // Refuse to validate anything if we have an invalid signer set.
+        if !self.is_valid() {
+            return Err(SignatureError::new());
+        }
+
         // If the signature contains less than the threshold number of signers or more
         // than the hardcoded limit, there's no point in trying.
         if multi_sig.signatures.len() < self.threshold as usize
```

Comment:
:wrench: Thinking this should get updated for the new `is_valid` check right before?
Should the MAX_SIGNATURES check get moved into `is_valid()`? Unsure on the MAX_SIGNATURES can see arguments both ways.

Reply:
This is checking a multi sig object, I don't see hows that related to is_valid (which checks the SignerSet)

Reply:
doh, you're right I read the `multi_sig` as `self`